cobc/flag.def

```diff
@@ -232,6 +232,16 @@ CB_FLAG (cb_listing_symbols, 1, "tsymbols",
 CB_FLAG (cb_listing_cmd, 1, "tcmd",
 	_("  -ftcmd                specify command line in listing"))
 
-CB_FLAG_ON (cb_diagnostic_show_option, 1, "diagnostics-show-option",
+CB_FLAG_ON (cb_diagnostics_show_option, 1, "diagnostics-show-option",
 	_("  -fno-diagnostics-show-option\tsuppress output of option that directly\n"
 	  "                        controls the diagnostic"))
+
+CB_FLAG (cb_diagnostics_show_caret, 1, "diagnostics-show-caret",
+	_("  -fdiagnostics-show-caret     display source context on warning/error diagnostic\n"
+	  "                               (the source is displayed without any REPLACE performed."
+	))
+
+CB_FLAG_ON (cb_diagnostics_show_line_numbers, 1, "diagnostics-show-line-numbers",
+	_("  -fno-diagnostics-show-line-numbers     suppress display of line numbers in diagnostics"
```

Comment:
"strange" thing to do, but please start the second line on the same position that is used for `diagnostics-show`... and as the name is long delimit the first line with \t instead of spaces